projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -211,4 +213,16 @@ public static function load_marketplace_products_updater() {
 	public static function load_first_posts_stream_helpers() {
 		require_once __DIR__ . '/features/first-posts-stream/first-posts-stream-helpers.php';
 	}
+
+	/**
+	 * Load Verbum Comments.
+	 */
+	public static function load_verbum_comments() {
+		if ( class_exists( 'Verbum_Comments' ) ) {
```

Comment:
class_exists condition will be removed when this is finalized, merged into WPCOM, removed from mu-plugins/verbum and we're ready to transfer over to Verbum_Comments inside of jetpack-mu-wpcom plugin.